### PR TITLE
🏎️ Added Forza-API

### PR DESCRIPTION
**Pull request by DaneeSkripter**
- [ ] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [ ] My addition is ordered alphabetically
- [ ] My submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column is padded with one space on either side
- [ ] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1185,6 +1185,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Motor Vehicle Department, Maharashtra, India](https://apisetu.gov.in/public/api/transportmh) | Driving License (DL) and Vehicle Registration Certificate (RC) of the State Maharashtra of India | `apiKey` | Yes | No |
 | [NHTSA](https://vpic.nhtsa.dot.gov/api/) | NHTSA Product Information Catalog and Vehicle Listing | No | Yes | Unknown |
 | [Smartcar](https://smartcar.com/docs/) | Lock and unlock vehicles and get data like odometer reading and location. Works on most new cars | `OAuth` | Yes | Yes |
+| [Forza-API](https://forza-api.tk) | Show random image of car from Forza | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1188]: Remove the `API` from the title and change the link to this one: https://images.forza-api.tk/

```suggestion
| [Forza](https://images.forza-api.tk) | Show random image of car from Forza | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @DaneeSkripter 😄 

Can you resolve this review, please?

Also, I think it's better to change this API to the [Games & Comics](https://github.com/public-apis/public-apis#games--comics) section and put it in alphabetical order. Can you do this please?

File: README.md (hunk 1 above)
Comment (DaneeSkripter) [reply README.md:1188]: Ok, I will do it.

**Comment by DaneeSkripter** [comment]
@matheusfelipeog done.

**Comment by DaneeSkripter** [comment]
Can anyone merge this pull request?

File: README.md [hunk 2]
```diff
@@ -1186,6 +1187,7 @@ API | Description | Auth | HTTPS | CORS |
 | [NHTSA](https://vpic.nhtsa.dot.gov/api/) | NHTSA Product Information Catalog and Vehicle Listing | No | Yes | Unknown |
 | [Smartcar](https://smartcar.com/docs/) | Lock and unlock vehicles and get data like odometer reading and location. Works on most new cars | `OAuth` | Yes | Yes |
 
+
```
Comment (matheusfelipeog) [inline README.md:1190]: Remove this blank space, please:

```suggestion
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Looks like you added a blank space, can you fix that?

File: README.md (hunk 2 above)
Comment (DaneeSkripter) [reply README.md]: Done, can you now merge this pull request?

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✅

Thanks for the update ;)

**Pull request merged**